examples/floret-demo.ipynb

```diff
@@ -0,0 +1,537 @@
+   "id": "75650836-3bd2-4d04-bd74-cb173e9a95a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install floret spacy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82d23b16-977e-49f9-a071-3ef28d64780e",
+   "metadata": {},
+   "source": [
+    "Let's start by downloading some vectors. These vectors can be found on [GitHub](https://github.com/explosion/spacy-vectors-builder/releases/tag/en-3.4.0). The code below downloads and unzips the vectors. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7a25a53-7fee-49f9-b4db-6b1d2d9f1bd2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! wget https://github.com/explosion/spacy-vectors-builder/releases/download/en-3.4.0/en_vectors_floret_md.floret.gz"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fa2beec-99fc-4335-868a-3a11335ef4a6",
+   "metadata": {},
+   "source": [
+    "Now that these vectors are downloaded, let's create a spaCy pipeline that has these vectors loaded. Note that we can simply pass the `.gz` file to `spacy init`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ee00a593-c328-4a15-84dc-2f52b1db9d44",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! spacy init vectors en en_vectors_floret_md.floret.gz en_vectors_floret_md --mode floret"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e6c2f90e-bd2c-4f4f-b6a0-b528dee10a12",
+   "metadata": {},
+   "source": [
+    "We now have a pipeline on disk called `en_core_floret_md`. Let's make some comparisons with the `en_core_web_md` model on disk. In order to do that we will first need to download it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5a32ffb5-87c0-4757-95d4-9f192381610e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! spacy download en_core_web_md"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "065fdb39-ab33-4893-931d-44f582fe907b",
+   "metadata": {},
+   "source": [
+    "Great, let's now load both pipelines."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1214ba7a-d8cc-4b2d-b607-8ee44a4d6e77",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import spacy\n",
+    "\n",
+    "# This is the standard spaCy pipeline\n",
+    "nlp_md = spacy.load(\"en_core_web_md\")\n",
+    "\n",
+    "# This is the spaCy pipeline with floret vectors\n",
+    "nlp_fl = spacy.load(\"en_vectors_floret_md\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "185f6e06-f324-44d8-9b1a-1299368fe35a",
+   "metadata": {},
+   "source": [
+    "As the blogpost explains, the floret embeddings are a bit different. Floret comes with subword embeddings instead of just word embeddings. It uses a hashing trick under the hood to keep things lightweight, but having these subwords at your disposal means that you can expect different behavior. \n",
```

Comment:
```suggestion
    "As the blog post explains, the floret embeddings are a bit different. floret comes with subword embeddings instead of just word embeddings. It uses a hashing trick under the hood to keep things lightweight, but having these subwords at your disposal means that you can expect different behavior. \n",
```